kinbot/kb.py

```diff
@@ -176,18 +185,47 @@ def main():
         logging.info('Starting the search for homolytic scission products')
         well0.homolytic_scissions = HomolyticScissions(well0, par, qc)
         well0.homolytic_scissions.find_homolytic_scissions()
-  
     # initialize the master equation instance
-    mess = MESS(par, well0)
-    mess.write_input()
-    #mesmer = MESMER(par, well0)
-    #mesmer.write_input()
```

Comment:
How is this affecting the connection to MESMER? I think we still want to maintain that option.

Reply:
OK, I misread this, these are deleted lines. Ignore this comment.